packages/p2p/src/components/my-profile/my-profile.jsx

```diff
@@ -105,14 +129,15 @@ const MyProfile = observer(() => {
                                         : '-'}
                                 </div>
                             </Table.Cell>
-                            <div className='my-profile__stats-cell-separator' />
+                            {!general_store.props.is_mobile && <div className='my-profile__stats-cell-separator' />}
```

Comment:
Please use `isMobile` from `@deriv/shared`, we're moving to that instead of using props.